linker/Linker/Driver.cs

```diff
@@ -118,6 +118,11 @@ void Run ()
 							continue;
 						}
 
+						if (token == "--used-attribute--types-only") {
```

Comment:
Is that a double dash in the middle?

Reply:
Noticed that last night as well.  Thanks.